src/github.com/getlantern/fronted/direct.go

```diff
@@ -136,6 +137,8 @@ func (d *direct) Do(req *http.Request) (*http.Response, error) {
 // Dial persistently dials masquerades until one succeeds.
 func (d *direct) Dial(network, addr string) (net.Conn, error) {
 	gotFirst := false
+	candidateCh := <-candidateChCh
+	candidateChCh <- candidateCh
```

Comment:
It's unclear to me what happens if Configure() is called concurrently to this.  It might be safer and easier to read to use an [atomic.Value](https://golang.org/pkg/sync/atomic/#Value).